.github/workflows/k8s-ci.yaml

```diff
@@ -0,0 +1,37 @@
+name: k8s CI
```

Comment:
So does this running telling us anything about whether what's here is safe to run https://github.com/pytorch/serve/tree/master/kubernetes - if not then I'm not sure this should be its own CI job

Reply:
The goal is to have a CI run with a k8s executing multiple test cases. Right now this has a single testcase similar to docker

If things work with minikube on one instance, then we can assume it will work with https://github.com/pytorch/serve/tree/master/kubernetes ? 

if not with minikube, we will have to implement something with multiple machines, aws etc. That might be validating EKS more than torchserve

Reply:
Makes sense - I think then It'b be good to check whether the kserve docker image works https://github.com/pytorch/serve/tree/master/kubernetes/kserve